app/views/_components/offer-panel/template.njk

```diff
@@ -253,11 +283,9 @@
 
 {% endif %}
 
-  {% if params.skeCondition %}
-    <h3 class="govuk-heading-s govuk-!-margin-top-6">
-      Other conditions
-    </h3>
-  {% endif %}
+  <h3 class="govuk-heading-s govuk-!-margin-top-6">
+    Safeguarding and other checks
```

Comment:
Are these just 'checks' or conditions? Similar to my other comment on the page before about other safeguarding checks, it seems like on the previous page these were split by: 1. Statutory conditions 2. Providing evidence of qualifications and then 3. Other safeguarding checks.

Wonder if on this page they should follow the same order and groupings? So 1. Academic conditions (which would include SKE info) 2. Statutory conditions 3. Evidence of qualifications 4. Other/Safeguarding conditions (and this is where they can add anything further if needed)?  Also maybe the statutory conditions should got first as those would be more NB, then academic etc... might need to test that out to see how it looks though!

Reply:
I didn't give that much thought to the groupings or labels yet which is probably why it's a bit inconsistent!

I wonder if we should use the same groupings on the provider-facing and candidate-facing interfaces, or if they should be slightly different?